ui/client/package.json

```diff
@@ -139,7 +139,7 @@
     "@cypress/browserify-preprocessor": "^3.0.1",
     "@cypress/code-coverage": "^3.9.7",
     "@hot-loader/react-dom": "^17.0.1",
-    "@pixability-ui/federated-types": "^0.1.0",
+    "@touk/federated-types": "1.0.0",
```

Comment:
How long do we plan to use own version ? Do we plan to populate our fixes/chagnes to pixability-ui codebase ?

Reply:
maybe, but it looks not well maintained
i don't belive in them. changes are needed and I'm prepared for maintenance here https://github.com/TouK/federated-types